### DISCUSSION TITLE
Willow vs willow-Whisper vs .....

### DISCUSSION THREAD

**Discussion by bert269** [toverainc/willow-inference-server#117] [General]
Ok - today I noticed on Github that there were some changes to WIS, so I stopped docker and do a re-install. During the process I niticed:
Cloning into 'models/tovera-wis-whisper-small......  -medium..... &  -large-v.

My question is, during the setup of a willow device, do I need to change my config from:
**...model=large&beam_size=5**
to :
**model=wis-whisper-large-v2&beam_size=5**

or do I leave it large? I know when it was only willow, the large worked for large-v2, but not sure about wis-whisper?

Thanks

**Comment by nikito** [comment]
It is always identified as large when referencing in the API. The model has actually always been large-v2, or at least has been that way for many iterations. 🙂
